benchmarks/README.md

```diff
@@ -27,3 +27,63 @@ Please refer to each subfolder to discover each benchmark suite
 
 * [Fast RNNs benchmarks](fastrnns/README.md)
 
+
+
+## PyTorch ASV benchmarks
+
+Benchmarking PyTorch with Airspeed Velocity.
+
+
+Usage
+-----
+
+Airspeed Velocity manages building and Python virtualenvs or conda envs by
+itself, unless told otherwise (e.g. with `--python=same`).
```

Comment:
A feature that I've always found extremely irritating about asv XD

Reply:
Yep, the default is quite annoying. It makes sense for CI or running it on a dedicated server, but not for typical use when you're developing. I've never complained about it, but maybe I should go open an issue now.

Reply:
Fixed now in ASV master, `asv dev` does this now.